templates/http-c/content/spin.toml

```diff
@@ -0,0 +1,15 @@
+spin_version = "1"
+authors = ["{{authors}}"]
+description = "{{project-description}}"
+name = "{{project-name}}"
+trigger = { type = "http", base = "{{http-base}}" }
+version = "0.1.0"
+
+[[component]]
+id = "{{project-name | kebab_case}}"
+source = "main.wasm"
```

Comment:
Should there be a .gitignore for this output?

Reply:
Added